### ISSUE TITLE
[BUG] Issue transferring certain FT to an EVM address from a Cadence account

### ISSUE THREAD

**Issue by JeffreyDoyle** [onflow/FRW#55]
### Current Behavior

Flow Wallet extension is unable to transfer certain fungible tokens to an EVM address from a Cadence account

### Expected Behavior

- Flow Wallet can successfully transfer any fungible token to/from a supported EVM address and a Cadence account

## Steps to Reproduce

- Aquire some st.FLOW tokens in a Cadence account on Flow Wallet extension
- Attempt to transfer these st.FLOW tokens to another account

## Expected Outcomes

- [ ] Flow Wallet can successfully transfer any fungible token to/from a supported EVM address and a Cadence account

## Platforms

- [ ] Extension

## Details:

![Image from iOS (1)](https://github.com/user-attachments/assets/12a20f13-4251-431b-a2e8-94cf7d1e1b8d)

**Comment by JeffreyDoyle** [comment]
Update:

This is also happening for users attempting to move stFLOW from their Cadence account to a COA.

They are seeing the following error:

```
There is an error in the cadence code of this transaction.
[Error Code: 1101] error caused by: 1 error occurred: * transaction execute failed: [Error Code: 1101] cadence runtime error: Execution failed: error: panic: Could not construct Vault type from identifier: A.5598c0652b899eb40f169dd5949bdbe0bf36ffde.undefined.Vault --> 57f027855c7904b5d41eea88e8156d695d45a6276cab229e52b8b50a6d969a70:45:15 | 45 | ?? panic("Could not construct Vault type from identifier: ".concat(vaultIdentifier)) | ^^^
```

**Comment by Peppermint1020** [comment]
Han fixed this issue in extension prod version 2.5.18
This one could fix the issue. the token info transfered in the above tx is taken from evm token list. This package will check the address first and get the identifier.
